.gitignore

```diff
@@ -9,6 +9,7 @@ bin
 artifacts
 testbin/*
 .DS_Store
+go.work
```

Comment:
Should we include go.work.sum as well? Also, should we add it to .dockerignore?

Reply:
Done.